modules/op-chains-generating-x509-secret.adoc

```diff
@@ -0,0 +1,55 @@
+// This module is included in the following assemblies:
+// * secure/using-tekton-chains-for-openshift-pipelines-supply-chain-security.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="chains-generating-x509-secret_{context}"]
+= Generating the x509 key pair using the TektonConfig CR
+
+To use the `x509` signing scheme for {tekton-chains} secrets, you must generate the `x509` key pair.
+
+You can generate the `x509` key pair by setting the `generateSigningSecret` field in the `TektonConfig` custom resource (CR) to `true`. 
+The {pipelines-title} Operator generates an `ecdsa` type key pair: `x509.pem` private key and `x509-pub.pem` public key, and stores them in the `signing-secrets` secret in the `openshift-pipelines` namespace.
```

Comment:
```suggestion
The {pipelines-title} Operator generates an `ecdsa` type key pair: an `x509.pem` private key and an `x509-pub.pem` public key. The Operator stores the keys in `signing-secrets` that exists in the `openshift-pipelines` namespace.
```

Reply:
i believe based on the guidelines the `signing-secrets` need a sepecification of what kinf of object it is: a secret. So I will keep that part as the `signing-secrets` secret. I applied the rest of the changes.